azure/exports/service.template

```diff
@@ -4,11 +4,14 @@ Wants=network-online.target sshd.service sshd-keygen.service
 After=network-online.target
 
 [Service]
-ExecStart=/usr/bin/python  -u /usr/sbin/waagent -daemon
+Environment=PYTHONPATH=$DESTDIR/rootfs/usr/lib/python2.7/site-packages/
+Environment=WALA_ROOT=$DESTDIR/rootfs
+ExecStart=/usr/bin/python  -u usr/sbin/waagent -daemon
 Restart=always
 RestartSec=5
-WorkingDirectory=${DESTDIR}/rootfs
-RuntimeDirectory=${NAME}
+WorkingDirectory=/var/lib/containers/atomic/wala-container.0/rootfs
+RuntimeDirectory=wala-container
```

Comment:
does the container use the runtime directory at all?  Could we just drop this line?